catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/XmlModelBuilder.java

```diff
@@ -324,6 +335,40 @@ public void setValue(JAXBElement<?> valueNode) {
     }
   }
 
+  private static class DistanceBufferSupplier extends TerminalNodeSupplier {
+    private Double distance = null;
+
+    DistanceBufferSupplier(final TerminalNodeSupplier original) {
+      super(original.reducer);
+      if (original.propertyNode != null) {
+        this.setProperty(original.propertyNode);
+      }
+      if (original.valueNode != null) {
+        this.setValue(original.valueNode);
+      }
+    }
+
+    public DistanceBufferSupplier setDistance(Double distance) {
+      notNull(distance);
+      this.distance = distance;
+      return this;
+    }
+
+    @Override
+    public JAXBElement<?> get() {
+      JAXBElement<?> reduced = super.get();
+      notNull(distance);
```

Comment:
✏️ Use the version of this that allows you to specify a message

https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/Validate.html#notNull(T,%20java.lang.String,%20java.lang.Object...)